### PR TITLE
Run the formatter

### PR DESCRIPTION
**Pull request by tcharding**
I don't want to wait for the bot, just run the formatter now.

No manual changes.

### CODE REVIEW

**Comment by Kixunil** [comment]
My displeasure is mainly about having to review things that don't seem priority to me and have very little value.

**Comment by apoelstra** [comment]
You could trust the PR title to mean "no changes here, nothing to review".

File: bitcoin/src/bip152.rs [hunk 1]
```diff
@@ -244,8 +244,10 @@ impl HeaderAndShortIds {
                 });
             } else {
                 match version {
-                    1 => short_ids.push(ShortId::with_siphash_keys(&tx.compute_txid(), siphash_keys)),
-                    2 => short_ids.push(ShortId::with_siphash_keys(&tx.compute_wtxid(), siphash_keys)),
+                    1 =>
+                        short_ids.push(ShortId::with_siphash_keys(&tx.compute_txid(), siphash_keys)),
+                    2 => short_ids
+                        .push(ShortId::with_siphash_keys(&tx.compute_wtxid(), siphash_keys)),
```
Comment (Kixunil) [inline bitcoin/src/bip152.rs:250]: FTR IMO this is harder to read and the previous was better. If it had curly braces at least it'd look OK.

**Review by Kixunil — APPROVED** [review]
ACK 40bee24de78348b77eec901e116be043035adde7

File: bitcoin/src/bip152.rs (hunk 1 above)
Comment (apoelstra) [reply bitcoin/src/bip152.rs:250]: Agreed, but shitty formatting is the price we pay to have consistent formatting in Rust.

Comment (Kixunil) [reply bitcoin/src/bip152.rs:250]: I'm still not convinced consistency is a goal worth pursuing. Although if there was a formatter that was flexible enough and a semantic merge tool we could get rid off of all these discussions and save a ton of time merging things.

Comment (tcharding) [reply bitcoin/src/bip152.rs:250]: Perhaps we should be discussing `rustfmt` at the summit (maybe over drinks since I doubt there is a good solution).

Comment (Kixunil) [reply bitcoin/src/bip152.rs:250]: If you can teleport me there and back then we can.

Comment (tcharding) [reply bitcoin/src/bip152.rs:250]: I'm already using my teleport machine for myself, its rubbish anyways - takes 14 hours to work and there are 300 other people on it.

**Pull request merged**